.github/workflows/bench.yml

```
@@ -0,0 +1,53 @@
+name: Bench
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  bench:
+    name: Benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          profile: minimal
+          toolchain: stable
+      - name: Chown cargo registry because of caching
+        run: sudo chown -R runner ~/.cargo/registry
+      - name: Cache cargo registry and target
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            target
+          key: bench-${{ steps.toolchain.outputs.rustc_hash }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Run benchmark
+        run: |
+          cargo bench
+      - name: Run benchmark
+        run: |
+          cargo bench | tee output.txt
+      - name: Store benchmark result
+        uses: rhysd/github-action-benchmark@v1
+        with:
+          tool: 'cargo'
+          output-file-path: output.txt
+#          skip-fetch-gh-pages: true
+          fail-on-alert: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-on-alert: true
+          alert-comment-cc-users: '@mgrachev'
```

Maybe better specify @dotenv-linter/core here?
```suggestion
          alert-comment-cc-users: '@dotenv-linter/core'
```